From u-boot  Mon Nov 01 04:43:47 2021
From: Yifeng Zhao <yifeng.zhao () rock-chips ! com>
Date: Mon, 01 Nov 2021 04:43:47 +0000
To: u-boot
Subject: [PATCH] dts: rockchip: rk3399: enable emmc phy for spl
Message-Id: <20211101044347.17822-1-yifeng.zhao () rock-chips ! com>
X-MARC-Message: https://marc.info/?l=u-boot&m=163574185622041

adapting commit ac804143cf ("mmc: rockchip_sdhci: add phy and clock
config for rk3399") to fix the issue "Not found emmc phy device".

Signed-off-by: Yifeng Zhao <yifeng.zhao@rock-chips.com>
---

 arch/arm/dts/rk3399-u-boot.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/dts/rk3399-u-boot.dtsi b/arch/arm/dts/rk3399-u-boot.dtsi
index 73922c328a..716b9a433a 100644
--- a/arch/arm/dts/rk3399-u-boot.dtsi
+++ b/arch/arm/dts/rk3399-u-boot.dtsi
@@ -88,6 +88,10 @@
 	u-boot,dm-pre-reloc;
 };
 
+&emmc_phy {
+	u-boot,dm-pre-reloc;
+};
+
 &grf {
 	u-boot,dm-pre-reloc;
 };
-- 
2.17.1


